thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -2043,6 +2065,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor
             }
         }
         while (!endOfInput);
+        stopReadAhead();
```

Comment:
Note to reviewer: stopReadAhead() might not have been called before is KJ::stop() was called and thus set endOfInput=true.